pinot-common/src/main/java/com/linkedin/pinot/common/utils/FileUploadUtils.java

```diff
@@ -14,13 +14,16 @@
  ******************************************************************************/
 package com.linkedin.pinot.common.utils;
 
+import com.google.common.util.concurrent.Uninterruptibles;
```

Comment:
Isn't it easier to just loop around N times in the sendSegmentFile() method? I would go to the extent of removing (or at least deprecating) the sendFile() method, and introducing another public method of sendSchemaFile() or somethign like that.
We need retries only for segments and not for schemas.